lib/src/components/NxTooltip/NxTooltip.tsx

```diff
@@ -36,7 +37,8 @@ function fixOptional(props: Props): TooltipProps {
     onClose: props.onClose || undefined,
     open: props.open === null ? undefined : props.open,
     placement: convertPlacement(props.placement),
-    title: props.title || ''
+    title: props.title || '',
+    'aria-label': textContent(props.title)
```

Comment:
This `fixOptional` function isn't really the right place to do this

Reply:
I wasn't able to duplicate with ChromeVox. That reader for me only reads the aria-label if I tab to the `<th>`, if I use the cv+arrow keys to navigate around it just stops on the name element and reads the element contents instead of the label when the NxTooltip is applied to the `<th>`.

In VO+Safari the reader reads out "name unsorted name" instead. I tried several different options but I can't seem to get any better behavior here.

Reply:
> I wasn't able to duplicate with ChromeVox. That reader for me only reads the aria-label if I tab to the <th>, if I use the cv+arrow keys to navigate around it just stops on the name element and reads the element contents instead of the label when the NxTooltip is applied to the <th>.

That's what I'm getting now too. Not sure what was different before.  However, this particular comment thread wasn't about behavior, it was about code organization and how this line does not fit the purpose of the function that it is in.

Reply:
5d513469f1c3e8bdbc9e91fa472b5fac4fbcab96